application/frontend/components/BaseRestController.php

```diff
@@ -36,7 +36,7 @@ public function actionIndex()
         $this->throwNotAllowed();
     }
 
-    public function actionUpdate()
+    public function actionUpdate($id)
```

Comment:
Should these be `$employeeId`? Since they're on the `BaseRestController`, I'm not sure.